go.sum

```diff
@@ -91,10 +91,17 @@ github.com/grailbio/base v0.0.6 h1:Hr1OHe16+sQEf4CytYlTu2bR+Sq5VENKA/sZoDBGo+k=
 github.com/grailbio/base v0.0.6/go.mod h1:OFVz7zmqb1D+Jbew0B4DCIpl4ozzVFxf+JKQZBBIQzE=
 github.com/grailbio/base v0.0.7 h1:f9qigc0yUuQRIMVr+EmWCLT9pTlVaLshrGTfrtQl3kE=
 github.com/grailbio/base v0.0.7/go.mod h1:VL8MWdM8WxkFUs4ribgWoGYlfty6Xyrat+lNNWWVCfs=
+github.com/grailbio/base v0.0.9 h1:Xv797SZiLcFE64hztiT9eZ4LSJjc6beZO8wmySIklyc=
+github.com/grailbio/base v0.0.9/go.mod h1:p8iBwwz1Qa84kSR7y1qVXFSmgexI9IVwQwCA32OkVec=
 github.com/grailbio/bigmachine v0.5.5 h1:uHdPrVTKw9BQmcfE70b7q126LJac8TiCnqOvS+UmvOo=
 github.com/grailbio/bigmachine v0.5.5/go.mod h1:8cYMHQBaSMyR9Gy9vh6YDE6uo+SgbEsnBXIGVp3gKGE=
 github.com/grailbio/bigmachine v0.5.6 h1:AVJ7tWEXRGqscjbQnr2uXi//JKKSxxJ8/xrm9dgY+Z8=
 github.com/grailbio/bigmachine v0.5.6/go.mod h1:cwLU340iN9dVoitv10KfDwkMBzhG/gGAgPOepRUUgIg=
+github.com/grailbio/bigmachine v0.5.7 h1:RaYi4wa4el62yqrw1qTB+KVmmlcS8VhDy59/l+8MMOk=
+github.com/grailbio/bigmachine v0.5.7/go.mod h1:wvOUthoZPxKKJ829ClWaO/uRTxaW3DLm7LyUQHO8ed0=
+github.com/grailbio/testutil v0.0.1/go.mod h1:j7teGaXqRY1n6m7oM8oy954lxL37Myt7nEJZlif3nMA=
+github.com/grailbio/testutil v0.0.3 h1:Um0OOTtYVvyxwQbO48K3t6lNmLPY4sL3Vn6Sw0srNy8=
+github.com/grailbio/testutil v0.0.3/go.mod h1:f9+y7xMXeXwyNcdV5cmo6GzRiitSOubMmqcqEON7NQQ=
```

Comment:
```suggestion
```

Reply:
For [reasons](https://github.com/grailbio/bigslice/commit/513f25a0c96aad6253aceef9a7a13c9dd56ce7e4), we do not add the `grailbio/base/testutil` checksums. If you remove these, the builds should run successfully.